app/controllers/api/auth/podcasts_controller.rb

```diff
@@ -1,11 +1,24 @@
 # encoding: utf-8
 
-class Api::Auth::PodcastsController < Api::PodcastsController
+class Api::Auth::PodcastsController < Api::BaseController
```

Comment:
so when I call super in show I am not calling the un-authed version that will redirect back to myself